src/SprykerEco/Yves/Ratepay/Theme/default/views/summary/summary.twig

```diff
@@ -0,0 +1,50 @@
+{% extends model('template') %}
+
+{% define data = {
+    quoteTransfer: required
+} %}
+
+{% block body %}
+    {% if data.quoteTransfer.payment.paymentMethod == 'INVOICE' %}
```

Comment:
Is it possible to move this logic outside template? Or this is for test purposes?

Reply:
preferably don't move logic to project level

Reply:
@alex-galych Can we fix this?